src/structures/Message.js

```diff
@@ -419,7 +419,7 @@ class Message extends Base {
   }
 
   /**
-   * Options that can be passed into editMessage.
+   * Options that can be passed into Message#edit.
```

Comment:
A link can be used here to help navigate to the `Message#edit` options on the docs.
```suggestion
   * Options that can be passed into {@link Message#edit}.
```